scopes/lanes/lanes/lanes.graphql.ts

```diff
@@ -50,52 +50,58 @@ export function lanesSchema(lanesMain: LanesMain): Schema {
         getLaneByName(name: String): LanesData
         getCurrentLaneName: String
         getDiff(values: [String], options: DiffOptions): GetDiffResult
+        getLaneComponents(name: String): [Component]
       }
 
       type Query {
-        lanes: Lanes
+        getLanes: [LanesData]
+        getLaneByName(name: String): LanesData
+        getCurrentLaneName: String
+        getLaneDiff(values: [String], options: DiffOptions): GetDiffResult
+        getLaneComponents(name: String): [Component]
```

Comment:
wut? why did you drop the top level grouping under "lanes"?
think of it from a consumer viewpoint

Reply:
should be something like this:
```gql
{
  ...
  lanes {
    currentLane: string;
    list: [string];
    diff(laneIds [string], options)
  }

  lane(id: "surfaces") { // could be array
    id
    author
    ...
    components (filters... ) { ... }
  }
}
```

Reply:
this is the new schema

```
    type Lane {
        id: String!
        isMerged: Boolean
        remote: String
        components: [Component!]!
        currentLane: Lane
        diff(toLaneId: String!, options: DiffOptions): GetDiffResult
      }

      type Query {
        lanes(id: String): [Lane!]!
      }
      ```

Reply:
ok, talked with @GiladShoham, let's go with this option:
```gql
type Lane {
  id: string
  isMerged: boolean
  remote: string
  components(filters): [Component]
  # ...
}

# automatically goes to the default "host" 
# nice to have: extends "scope" and "workspace" to have "lanes" property
query lanes {
  # ids filter list to include only these ids
  list(ids: [LaneId], offset, limit) [lane] # check if we can return a single lane when requesting a single lane
  diff(from: LaneId, to: LaneId, options): LaneDiff
  current: Lane, # note it is `current` and not `currentLane`
}
```